playbooks/maas-managed-k8.yml

```diff
@@ -0,0 +1,154 @@
+---
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+- name: Gather facts
+  hosts: hosts
+  gather_facts: "{{ gather_facts | default(true) }}"
+  pre_tasks:
+    - include: "common-tasks/maas_excluded_regex.yml"
+    - name: Set the current group
+      set_fact:
+        maas_current_group: k8_all
+  tasks:
+    - name: Check if auth DB exists
```

Comment:
@xgerman I think we chatted about this in passing once, but I would really like to avoid having other clients (like these Ansible playbooks) interacting directly with the kubernetes-auth database. These plays should use the kubernetes-auth API instead.

Reply:
Refactored